src/main/java/nextstep/ladder/model/Ladder.java

```diff
@@ -0,0 +1,44 @@
+package nextstep.ladder.model;
+
+import nextstep.ladder.exception.MinimumException;
+
+import java.util.List;
+import java.util.Optional;
+
+public class Ladder {
+
+    private static final String MINIMUM_PEOPLE_MESSAGE = "참여자 수는 최소 2인 이상입니다.";
+    private static final String MINIMUM_LINE_MESSAGE = "사다리의 행 길이는 최소 2인 이상입니다.";
+    private static final int MINIMUM_ROW_LENGTH = 2;
+    private final List<Person> people;
```

Comment:
상수와 인스턴스 변수의 구분을 위해 공백 라인을 추가해보면 어떨까요?
```suggestion
    private static final int MINIMUM_ROW_LENGTH = 2;
    
    private final List<Person> people;
```